src/backend/cpu/kernel/regions.hpp

```diff
@@ -178,6 +178,9 @@ void regions(Array<T> out, const Array<char> in, af_connectivity connectivity)
         }
     }
 
+    for (auto& mapElem: lmap)
+        delete mapElem.second;
```

Comment:
I thought about unique pointers, but then i thought why add another abstraction instead of just cleaning them off at the end since this object's definition and usage is local to that function. Will using unique pointers make it more efficient ?